src/main/java/org/geysermc/discordbot/listeners/ErrorAnalyzer.java

```diff
@@ -17,13 +18,11 @@
 import java.util.regex.Pattern;
 
 public class ErrorAnalyzer extends ListenerAdapter {
-    private final Map<String, String> issueResponses;
     private final Map<Pattern, String> logUrlPatterns;
 
     private static final Pattern BRANCH_PATTERN = Pattern.compile("Geyser .* \\(git-[0-9a-zA-Z]+-([0-9a-zA-Z]{7})\\)");
 
     public ErrorAnalyzer() {
-        issueResponses = TagsManager.getIssueResponses();
```

Comment:
Doesn't this just kind of make it take more work in the future if we want to change the source of issue responses?

Reply:
Yes but it gives more flexibility for later to reload tags while we are running as its not caching it again when the class is created.